saleor/checkout/utils.py

```diff
@@ -871,6 +876,19 @@ def create_order(checkout: Checkout, tracking_code: str, discounts, taxes):
 
     # assign checkout payments to the order
     checkout.payments.update(order=order)
+
+    # remove checkout after order is created
+    checkout.delete()
+
+    # Create the order placed and email confirmation sent events
+    OrderEventManager()\
```

Comment:
A minor thing but we rather don't use backslashes to break long lines. Instead of chaining method calls here we could also just call them one by one.